doc/ref_arch/openstack/chapters/chapter04.md

```diff
@@ -143,8 +143,14 @@ CPU reservation for host: 1 core per NUMA
     - minimum: two nodes per profile
 -	HW specifications
     -	Boot disks are dedicated with Flash technology disks
-    - In case of DPDK usage, cores (together with their sibling threads) must be reserved for DPDK Poll Mode Drivers (PMD), hugepages set to at least 1 GB, and all power settings disabled in the BIOS. The number of cores reserved for this overhead depends on the desired throughput (the number of Mpps we want to deliver).
-    
+
+- In case of DPDK usage:
+
+| Layer | Description |
+|-------------|--|
+| Cloud infrastructure | Important is placement of NICs to get to balanced IO system, and configuration of NIC features. Server BIOS and Host OS kernel command line settings are described on DPDK.org in release notes and performance reports. Disabling power settings (like Intel Turbo Boost Technology) brings stable performance results, although understanding if and when they benefit workloads and enabling them can achieve better performance results. |
```

Comment:
Can you please add links to the release notes and performance reports in DPDK.org?

Reply:
Done in #1402. (I cannot Edit this one, will do here if that changes)